docs/components/DocumentationFooter.tsx

```diff
@@ -35,7 +33,7 @@ function githubUrl(path) {
 
   let pathAsMarkdown = path.replace(/\/$/, '') + '.md';
   if (pathAsMarkdown.startsWith('/versions/latest')) {
-    pathAsMarkdown = pathAsMarkdown.replace('/versions/unversioned');
+    pathAsMarkdown = pathAsMarkdown.replace('/versions/latest', '/versions/unversioned');
```

Comment:
Can we use the `{ LATEST_VERSION } from '~/constants/versions'` here? I think that might be more "semantically correct"

Reply:
But `LATEST_VERSION` contains numbered version, and here we probably need the exact string `latest`

Reply:
I think the path as markdown is a translation between URL and repo file/folder location, right? What we currently do for `latest` is copy over the exact "latest" versioned doc, to `pages/versions/latest`. This is not within the repository, it's excluded and only happens locally.

That's why I think we need that exact latest version string instead 😄  We also have `unversioned`, but that is technically the version which is not published. So I think this `latest` -> `unversioned` is not correct too. Right now, we released SDK 39, so `latest` should point at `SDK 39` imho.

Reply:
Yes, it translates URL -> repo file.

Reply:
Yes, it translates URL -> repo file. Currently, when `/versions/latest` is a _copy_ of `/versions/v39.0.0`, it behaves this way:
- `/versions/latest` in URL -> `/versions/unversioned` on GH
- `/versions/v39.0.0` in URL -> `/versions/v39.0.0` in repo
- `/versions/v38.0.0` and earlier -> no change (same as 39)

So, until URLs distinguish between `v39.0.0` and `latest`, we should redirect both of them to `unversioned`.